### PR TITLE
i18n(es): Update `frontmatter.md`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Updated `frontmatter.md`

### CODE REVIEW

File: docs/src/content/docs/es/reference/frontmatter.md [hunk 1]
```diff
@@ -258,4 +258,30 @@ title: Página para ocultar de la barra lateral autogenerada
...
+title: Página con una insignia
+sidebar:
+  # Usa la variante predeterminada que coincide con el color de acento de tu sitio
+  badge: Nuevo
+---
+```
+
+```md
+---
+title: Page with a badge
```
Comment (kevinzunigacuellar) [inline docs/src/content/docs/es/reference/frontmatter.md:280]:
```suggestion
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Thanks @dreyfus92. I left a couple suggestions 🚀

**Comment by dreyfus92** [comment]
Thank you so much for your review @kevinzunigacuellar 😁

File: docs/src/content/docs/es/reference/frontmatter.md [hunk 2]
```diff
@@ -258,4 +258,29 @@ title: Página para ocultar de la barra lateral autogenerada
 sidebar:
   hidden: true
 ---
+
+#### `badge`
+
+**tipo:** <code>string | <a href="/reference/configuration/#badgeconfig">BadgeConfig</a></code>
+
+Agrega una insignia a la página en la barra lateral cuando se muestra en un grupo de enlaces generado automáticamente.
+Cuando se usa una string, la insignia se mostrará con el color de acento predeterminado.
```
Comment (Waxer59) [inline docs/src/content/docs/es/reference/frontmatter.md:267]:
```suggestion
Cuando se usa un string, la insignia se mostrará con el color de acento predeterminado.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

File: docs/src/content/docs/es/reference/frontmatter.md [hunk 3]
```diff
@@ -258,4 +258,29 @@ title: Página para ocultar de la barra lateral autogenerada
 sidebar:
   hidden: true
 ---
+
```
Comment (delucis) [inline docs/src/content/docs/es/reference/frontmatter.md:261]: Looks like we’re missing the closing backticks for this code snippet, so the badge section is getting wrapped in a code block:
````suggestion
```

````

**Review by delucis — COMMENTED** [review]
Thanks @dreyfus92! Noticed one small bug, then LGTM.

**Comment by dreyfus92** [comment]
Thanks @delucis 😁

File: docs/src/content/docs/es/reference/frontmatter.md [hunk 4]
```diff
@@ -259,3 +259,29 @@ sidebar:
   hidden: true
 ---
 ```
+
+#### `badge`
+
+**tipo:** <code>string | <a href="/reference/configuration/#badgeconfig">BadgeConfig</a></code>
```
Comment (delucis) [inline docs/src/content/docs/es/reference/frontmatter.md:265]: Ooh, also noticed this link. Will get this fixed real quick and then LGTM. Thanks again @dreyfus92.

```suggestion
**tipo:** <code>string | <a href="/es/reference/configuration/#badgeconfig">BadgeConfig</a></code>
```

**Pull request merged**